x-pack/test/alerting_api_integration/common/config.ts

```diff
@@ -97,7 +98,7 @@ export function createTestConfig(name: string, options: CreateTestConfigOptions)
         ...xPackApiIntegrationTestsConfig.get('kbnTestServer'),
         serverArgs: [
           ...xPackApiIntegrationTestsConfig.get('kbnTestServer.serverArgs'),
-          '--server.publicBaseUrl=https://localhost:5601',
+          ...(options.publicBaseUrl ? ['--server.publicBaseUrl=https://localhost:5601'] : []),
```

Comment:
changed so `security_and_spaces` runs WITH the configured url, but `spaces_only` does not